Cargo.toml

```diff
@@ -87,35 +87,35 @@ serde_yaml = "0.9.34"
-aptos-language-e2e-tests = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-mempool = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-proptest-helpers = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-sdk = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-state-view = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-storage-interface = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-temppath = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-types = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-vm = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-vm-genesis = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-vm-logging = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-logger = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
-aptos-vm-types = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "20b44f6231514da8db6fad46a5e30c950464754c" }
+aptos-db = { git = "https://github.com/movementlabsxyz/aptos-core.git", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-executor = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-executor-test-helpers = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-executor-types = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-faucet-core = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-framework = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-language-e2e-tests = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-mempool = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-proptest-helpers = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
+aptos-sdk = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "d41e9c0c35ce563a3f3b070656abddeb994988e8" }
```

Comment:
Is this`monza`, should there be an associated PR.

Reply:
The changes have already been merged to `monza`.